src/main/scala/regmapper/RegField.scala

```diff
@@ -141,9 +142,29 @@ case class RegField(width: Int, read: RegReadFn, write: RegWriteFn, desc: Option
 {
   require (width > 0, s"RegField width must be > 0, not $width")
 
-  def pipelined = !read.combinational || !write.combinational
+  def pipelined: Boolean = !read.combinational || !write.combinational
```

Comment:
why are we randomly adding the type to some functions/vars and not others? This seems unrelated to the PR

Reply:
Rocket-chip is not run against the lint rules and does not pass lint. 

Public functions without a return type are a lint violation.

The return types were not added randomly. They were added to functions that were being modified. 

The Intellij IDE considers lint violations to be errors and this causes problems when using the IDE.

Henry suggested making a separate PR for rocket-chip lint fixes.

I can remove the Lint fixes and put them in a PR.

Reply:
Sounds good... could address this ancient issue: https://github.com/freechipsproject/rocket-chip/issues/255

Reply:
I don't think we should make lint changes *just* based on what your editor considers a linting issue. If we don't unilaterally enforce lint at the CI level, then anyone not using the same editor as you may just continue to commit code that your editor considers to be a lint violation. If you want to make style changes, then please add a CI check to lint the code.

Reply:
There is no scalastyle-config.xml in rocket-chip.